src/iuse.h

```diff
@@ -9,6 +9,7 @@
 #include "clone_ptr.h"
 #include "units.h"
 
+class Character;
```

Comment:
This could be a forward declaration

Reply:
isn't that what this is?

Reply:
Uh... yeah please excuse my addled brain.